WSSiOS/WSSiOS/Source/Presentation/Memo/MemoView/FeedEditView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  FeedEditView.swift
+//  WSSiOS
+//
+//  Created by Hyowon Jeon on 6/29/24.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+final class FeedEditView: UIView {
+    
+    //MARK: - Components
+    
+    let backButton = UIButton()
+    let completeButton = UIButton()
+    let scrollView = UIScrollView()
+    private let contentView = UIStackView()
+    let feedCategoryView = FeedCategoryView()
+    let feedContentView = FeedContentView()
+    let feedNovelConnectView = FeedNovelConnectView()
+    
+    //MARK: - Life Cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setUI()
+        setHierarchy()
+        setLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    //MARK: - UI
+    
+    private func setUI() {
+        self.do {
+            $0.backgroundColor = .wssWhite
+        }
+        
+        backButton.do {
+            $0.setImage(.icNavigateLeft.withRenderingMode(.alwaysOriginal), for: .normal)
+        }
```

Comment:
W1
Figma에서 확인해보면 백 버튼 색이 다릅니다! 모든 백 버튼 색깔별로 에셋을 넣을 수 없으니 이렇게 alwaysTemplate 모드로 가져온 뒤 tint 컬러로 색을 넣어주는 건 어떨까요?

```suggestion 
backButton.do {
    $0.setImage(.icNavigateLeft.withTintColor(.wssWhite, renderingMode: .alwaysTemplate),
                            for: .normal)
}
```

Reply:
예리하군